2017-04-08T09:05:23.443+0800 I CONTROL  [initandlisten] MongoDB starting : pid=4976 port=12345 dbpath=data 64-bit host=zhibinzhang
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] targetMinOS: Windows 7/Windows Server 2008 R2
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] db version v3.4.2
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] git version: 3f76e40c105fc223b3e5aac3e20dcd026b83b38b
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2h  3 May 2016
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] allocator: tcmalloc
2017-04-08T09:05:23.444+0800 I CONTROL  [initandlisten] modules: none
2017-04-08T09:05:23.445+0800 I CONTROL  [initandlisten] build environment:
2017-04-08T09:05:23.445+0800 I CONTROL  [initandlisten]     distmod: 2008plus-ssl
2017-04-08T09:05:23.445+0800 I CONTROL  [initandlisten]     distarch: x86_64
2017-04-08T09:05:23.445+0800 I CONTROL  [initandlisten]     target_arch: x86_64
2017-04-08T09:05:23.445+0800 I CONTROL  [initandlisten] options: { config: "conf/mongod.conf", net: { port: 12345 }, storage: { dbPath: "data" }, systemLog: { destination: "file", path: "logs/mongod.log" } }
2017-04-08T09:05:23.445+0800 W -        [initandlisten] Detected unclean shutdown - data\mongod.lock is not empty.
2017-04-08T09:05:23.449+0800 I -        [initandlisten] Detected data files in data created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2017-04-08T09:05:23.449+0800 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2017-04-08T09:05:23.449+0800 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=1527M,session_max=20000,eviction=(threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-04-08T09:05:25.731+0800 I CONTROL  [initandlisten] 
2017-04-08T09:05:25.731+0800 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-04-08T09:05:25.731+0800 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-04-08T09:05:25.732+0800 I CONTROL  [initandlisten] 
2017-04-08T09:05:26.706+0800 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory 'data/diagnostic.data'
2017-04-08T09:05:26.709+0800 I NETWORK  [thread1] waiting for connections on port 12345
2017-04-08T09:05:27.042+0800 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2017-04-08T09:06:57.662+0800 I NETWORK  [thread1] connection accepted from 127.0.0.1:56927 #1 (1 connection now open)
2017-04-08T09:06:57.688+0800 I NETWORK  [conn1] received client metadata from 127.0.0.1:56927 conn1: { driver: { name: "nodejs", version: "2.2.22" }, os: { type: "Windows_NT", name: "win32", architecture: "x64", version: "6.3.9600" }, platform: "Node.js v7.4.0, LE, mongodb-core: 2.1.7" }
2017-04-08T17:32:15.243+0800 W FTDC     [ftdc] Uncaught exception in 'FileRenameFailed: ¾Ü¾ø·ÃÎÊ¡£' in full-time diagnostic data capture subsystem. Shutting down the full-time diagnostic data capture subsystem.
